app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(players: list) -> int:
+    return sum((player.get_rating() for player in players))
```

Comment:
There is no need to pass a generator to `sum`, you could pass a generator expression as indicated[ in the checklist](https://github.com/yura-push/py-elves-and-dwarves/blob/develop/checklist.md#code-efficiency).
```suggestion
    return sum(player.get_rating() for player in players)
```